client/src/components/banner.jsx

```diff
@@ -92,29 +165,63 @@ class Banner extends React.Component {
     view.scrollIntoView({behavior: 'smooth'});
   }
   render() {
-    let { hash, scrollingDown } = this.state;
+    let { hash, scrollingDown, showShop, showSport, showStories, showActivism } = this.state;
     let buttonText = hash.split('&')[0].split('=')[1];
     return (
-      <NavbarContainer id="banner" scrollingDown={scrollingDown}>
-        <Navbar>
-        <LinkContainer>
-          <Links href=''>{this.props.titleData.categoryName}</Links>
-          <Links href=''> > {this.props.titleData.title}</Links>
-        </LinkContainer>
-        <BuyContainer>
-          <BuyInfo onClick={()=> this.scrollTo("reviews")}>{this.props.count} Reviews</BuyInfo>
-          <BuyInfo onClick={() => document.getElementById('guide').click()}>Size{' & '}Fit</BuyInfo>
-          <BuyInfo onClick={()=> this.scrollTo("impact")}>Impact</BuyInfo>
-          <BuyButton
-            onClick={buttonText === 'false' ? () => document.getElementById('buy').click() : () => this.scrollTo('inventory')}
-            buy={buttonText}
-          >
-          {buttonText === 'false' ? 'Add to Bag' : 'Buy'}
-          </BuyButton>
-        </BuyContainer>
-        </Navbar>
-  </NavbarContainer >
-          )
+      <>
+        <BannerContainer id="banner" scrollingDown={scrollingDown}>
+          <DropBanner>
+            <LinkContainer>
+              <Links href=''>{this.props.titleData.categoryName}</Links>
```

Comment:
if you wanted to deconstruct your props just for fun, you could. but also if you dont want to, thats fine too.